src/styles/untils/_extended.scss

```diff
@@ -0,0 +1,3 @@
+%margin {
+  margin: 0 auto;
+}
```

Comment:
I don't think this placeholder can be really useful, it's just a single margin rule
It's great to use placeholders, but don't overuse them